### PR TITLE
Minor Code Cleanup 

### PR DESCRIPTION
**Pull request by ajayk**
Minor Code Cleanup

### CODE REVIEW

**Comment by juherr** [comment]
:+1:

File: src/main/java/org/testng/TestNG.java [hunk 1]
```diff
@@ -276,7 +276,8 @@ public void initializeSuitesAndJarFile() {
     	//to parse the suite files (<suite-file>), if any
     	for (XmlSuite s: m_suites) {
 	      List<String> suiteFiles = s.getSuiteFiles();
-  			for (int i = 0; i < suiteFiles.size(); i++) {
+  			int suiteFilesSize = suiteFiles.size();
+			for (int i = 0; i < suiteFilesSize; i++) {
```
Comment (wengm) [inline src/main/java/org/testng/TestNG.java]: use enhanced for loop: for-each loop

File: src/main/java/org/testng/TestNGAntTask.java [hunk 2]
```diff
@@ -869,7 +869,7 @@ protected ExecuteWatchdog createWatchdog() /*throws BuildException*/ {
   protected void validateOptions() throws BuildException {
     int suiteCount = getSuiteFileNames().size();
```
Comment (wengm) [inline src/main/java/org/testng/TestNGAntTask.java]: can also use isEmpty()

**Comment by juherr** [comment]
@wengm The PR is 6 years old and is not sync at all with the master. You should fork the origin branch or just cherry-pick commit on a new branch and make a PR.

**Comment by krmahadevan** [comment]
Closing this PR since it needs to be revived before anything can be done

**Pull request closed without merging**